tests/test_mab.py

```diff
@@ -11,6 +13,45 @@
 
 class MABTest(BaseTest):
 
+    #################################################
+    # Test property decorator methods
+    ################################################
+    def test_learning_policy_property(self):
+        for lp in BaseTest.lps:
+            mab = MAB([1, 2], lp)
+            test_lp = mab.learning_policy
+            self.assertTrue(type(test_lp) is type(lp))
```

Comment:
anything else that we can assert other just the type?

Reply:
Not generically in a loop since each value in the different tuples has a different name

Reply:
I have added two more unit tests for this